packages/ember-routing/lib/system/route.js

```diff
@@ -1703,7 +1703,8 @@ let Route = EmberObject.extend(ActionHandler, Evented, {
     model supplied by the `model` hook.
 
     By default, the `setupController` hook sets the `model` property of
-    the controller to the `model`.
+    the controller to the `model`. If `model` is `undefined`, the default
```

Comment:
Instead of stating the default then correcting it with what happens when model is `undefined`, can we tweak how we state the default?

Perhaps something like:

> By default, the `setupController` hook sets the `model` property of the controller to the `model` specified when it is not `undefined`.